ui/page/wallet_page.go

```diff
@@ -129,6 +129,10 @@ func (pg *WalletPage) ID() string {
 }
 
 func (pg *WalletPage) OnResume() {
+	pg.loadWalletAndAccounts()
+}
+
+func (pg *WalletPage) loadWalletAndAccounts() {
```

Comment:
Should probably protect this with a mutex and instances where `pg.listItems` is accessed.